src/containers/AccountConnection.jsx

```diff
@@ -50,12 +50,12 @@ class AccountConnection extends Component {
     const folderField = connector && connector.fields && connector.fields.folderPath
     if (!folderField) return null
     const auth = account && account.auth
-    if (!folderField.default && auth && !auth.folderPath) {
-      return t('account.config.default_folder', connector)
-    }
     if (folderField.default) {
       return folderField.default
     }
+    if (!(auth && auth.folderPath)) {
+      return t('account.config.default_folder', connector)
+    }
     if (auth && auth.folderPath) {
       return account.folderPath
```

Comment:
You're right, I think that should, if there is an `auth` and `auth.folderPath`, we should trust that. If the user update the folderPath, it will update the auth in the related account and we will fallback here after the update. It seems more logical to me after thinking.